testplan/exporters/testing/base.py

```diff
@@ -191,6 +192,6 @@ def save_attachments(report: TestReport, directory: str) -> Dict[str, str]:
             dst_path = os.path.join(directory, dst)
             makedirs(os.path.dirname(dst_path))
             copyfile(src=src, dst=dst_path)
-            moved_attachments[dst] = dst_path
+            moved_attachments[dst] = os.path.join(".", ATTACHMENTS, dst)
```

Comment:
if we need a relative path here, maybe change dst_path to relative path?

Reply:
I think not necessary. `os.path.join(".", ATTACHMENTS, dst)` is a string written into json report. If we actually copy a file, using the absolute path is much reliable that it does not matter where to execute script.

Reply:
But move_attachements[dst] shall be the actual place where we copy attachment to.